src/loader/bgw_launcher.c

```diff
@@ -529,13 +573,13 @@ launcher_handle_message(HTAB *db_htab)
 	switch (message->message_type)
 	{
 		case START:
-			message_start_action(db_htab, message, vxid);
+			bgw_message_send_ack(message, message_start_action(db_htab, message, vxid));
```

Comment:
FWIW, I actually preferred the way it was when you set a return variable and had a `bgw_message_send_ack()` below. That avoids nesting functions, and makes it clear the result returned by these functions.